perf/stability/alertmanager/values.yaml

```diff
@@ -0,0 +1,57 @@
+image:
+  hub: quay.io/prometheus/alertmanager
+  tag: v0.20.0
+  pullPolicy: IfNotPresent
+
+replicaCount: 2
+config:
+  global:
+    resolve_timeout: 5m
+  route:
+    group_by: ['alertname', 'cluster']
+    group_wait: 10s
+    group_interval: 10s
+    repeat_interval: 1h
+    receiver: 'test'
+  receivers:
+    - name: 'test'
+      webhook_configs:
+        - url: 'http://am-webhook.istio-prometheus:5001/webhook'
+#      [Optional Slack Config] Need to replace <slack_webhook_url> with actual value first
+#      slack_configs:
+#        - api_url: <slack_webhook_url>
+#          channel: "#stability-test"
+#          send_resolved: true
+#          title: |-
+#            [{{ .Status | toUpper }}{{ if eq .Status "firing" }}:{{ .Alerts.Firing | len }}{{ end }}] {{ .CommonLabels.alertname }} for {{ .CommonLabels.job }}
```

Comment:
you cannot have templates in values right?
This is commented out, but please update comments or move to a separate file before merge.

Reply:
It can actually, with this block {{ toYaml .Values.config | b64enc | quote }}